src/patternfly/components/Select/select.scss

```diff
@@ -144,6 +145,17 @@
   background-color: var(--pf-c-select__toggle--BackgroundColor);
   border: none;
 
+  &.pf-m-disabled,
+  &:disabled {
+    pointer-events: none;
+
+    --pf-c-select__toggle--BackgroundColor: var(--pf-c-select__toggle--disabled--BackgroundColor);
```

Comment:
Can you move the variable to the top? We typically list variables before declarations.